app/js/questions_controller.js

```diff
@@ -43,6 +45,29 @@
     var tab_panels = container.querySelectorAll('[role="tabpanel"]');
     toggle_state(tab_panels, 'aria-hidden');
 
+    load_active_questions();
```

Comment:
Instead, we can do something like calling a function `tab_active(tab.parentNode.id)` and tab_active takes care of loading questions if they haven't been loaded yet.